src/pages/flight-plan/flight-plan.html

```diff
@@ -103,6 +103,21 @@ <h2>
 				</taf-list-item>
 			</ion-list>
 
+		</ion-list-header>
+		<ion-list-header>
+			<button ion-item (click)="onToggleSection(sections.airsigmets)" detail-none
+					[ngClass]="{'section-active': sections.airsigmets.open, 'section': !sections.airsigmets.open}">
+				<ion-icon item-left name="arrow-forward" *ngIf="!sections.airsigmets.open"></ion-icon>
+				<ion-icon item-left name="arrow-down" *ngIf="sections.airsigmets.open"></ion-icon>
+				{{ sections.airsigmets.title }}
+			</button>
+
+			<ion-list *ngIf="sections.airsigmets.open">
+				<ion-item *ngFor="let met of airsigmets">
```

Comment:
Empty item.

Reply:
You are correct this is empty currently! This is only a placeholder for when I get the AIRMET information filled in here.